package.json

```diff
@@ -5,17 +5,18 @@
   "workspaces": {
     "packages": [
       "packages/hardhat",
```

Comment:
Did you see the comment here about hardhat?

https://github.com/scaffold-eth/scaffold-eth-2/pull/353#pullrequestreview-1464498252

(I think you also missed some comments from the last review)

Reply:
thats for `scripts`, shall i remove hardhat completely as a package ? @carletex

Reply:
From https://github.com/scaffold-eth/scaffold-eth-2/pull/353#pullrequestreview-1464498252

> So another change: let's remove the "hardhat" package here.

So, yes, let's remove hardhat entirely here. This would be what people get after the choose "Foundry" in the CLI.